src/apps/dashboard/AppLayout.tsx

```diff
@@ -19,82 +19,61 @@ interface AppLayoutProps {
     drawerlessPaths: string[]
 }
 
-interface DashboardAppSettings {
-    isDrawerPinned: boolean
-}
-
-const DEFAULT_APP_SETTINGS: DashboardAppSettings = {
-    isDrawerPinned: false
-};
-
 const AppLayout: FC<AppLayoutProps> = ({
     drawerlessPaths
 }) => {
-    const [ appSettings, setAppSettings ] = useLocalStorage<DashboardAppSettings>('DashboardAppSettings', DEFAULT_APP_SETTINGS);
-    const [ isDrawerActive, setIsDrawerActive ] = useState(appSettings.isDrawerPinned);
+    const [ isDrawerActive, setIsDrawerActive ] = useState(false);
     const location = useLocation();
-    const theme = useTheme();
     const { user } = useApi();
 
-    const isDrawerAvailable = !drawerlessPaths.some(path => location.pathname.startsWith(`/${path}`));
-    const isDrawerOpen = isDrawerActive && isDrawerAvailable && Boolean(user);
-
-    useEffect(() => {
-        if (isDrawerActive !== appSettings.isDrawerPinned) {
-            setAppSettings({
-                ...appSettings,
-                isDrawerPinned: isDrawerActive
-            });
-        }
-    }, [ appSettings, isDrawerActive, setAppSettings ]);
+    const isSmallScreen = useMediaQuery((t: Theme) => t.breakpoints.up('sm'));
+    const isDrawerAvailable = Boolean(user)
+        && !drawerlessPaths.some(path => location.pathname.startsWith(`/${path}`));
+    const isDrawerOpen = isDrawerActive && isDrawerAvailable;
 
     const onToggleDrawer = useCallback(() => {
         setIsDrawerActive(!isDrawerActive);
     }, [ isDrawerActive, setIsDrawerActive ]);
 
     return (
         <Box sx={{ display: 'flex' }}>
-            <ElevationScroll elevate={isDrawerOpen}>
+            <ElevationScroll elevate={false}>
                 <AppBar
                     position='fixed'
-                    sx={{ zIndex: (muiTheme) => muiTheme.zIndex.drawer + 1 }}
+                    sx={{
+                        width: {
+                            xs: '100%',
+                            sm: `calc(100% - ${DRAWER_WIDTH}px)`
+                        },
```

Comment:
```suggestion
                        width: {
                            xs: '100%',
                            ...(isDrawerOpen ? { sm: `calc(100% - ${DRAWER_WIDTH}px)` } : {})
                        },
```

Reply:
I believe fixes https://github.com/jellyfin/jellyfin-web/pull/5020#issuecomment-1877924295, testing now.

Reply:
Good catch... I applied the same change that is used in the experimental app layout to fix.